sample-kotlin/src/main/java/com/fblitho/lithoktsample/errors/ListRowComponentSpec.kt

```diff
@@ -25,17 +28,17 @@ object ListRowComponentSpec {
 
   @OnCreateLayout
   fun onCreateLayout(c: ComponentContext, @Prop row: ListRow): Component =
-      Column.create(c)
-          .paddingDip(YogaEdge.VERTICAL, 8f)
-          .paddingDip(YogaEdge.HORIZONTAL, 32f)
-          .child(
-              Card.create(c)
-                  .content(
-                      Column.create(c)
-                          .marginDip(YogaEdge.ALL, 32f)
-                          .child(TitleComponent.create(c).title(row.title))
-                          .child(PossiblyCrashingSubTitleComponent.create(c).subtitle(row.subtitle))
-                          .build())
-                  .build())
-          .build()
+      column(c) {
+        paddingDip(YogaEdge.VERTICAL, 8f)
+        paddingDip(YogaEdge.HORIZONTAL, 32f)
+        children {
+          card {
+            content(column(c) {
+              marginDip(YogaEdge.ALL, 32f)
```

Comment:
Since we have named arguments in Kotlin I wonder if it would make sense to add an extension function that would make this line look like this:
`marginDip(all=32f)`